src/lib/tabwidget/tabwidget.cpp

```diff
@@ -194,6 +194,15 @@ void TabWidget::loadSettings()
     m_newEmptyTabAfterActive = settings.value("newEmptyTabAfterActive", false).toBool();
     settings.endGroup();
 
+    settings.beginGroup("Web-Browser-Settings");
```

Comment:
I think it's expected behavior when it change default zoom factor to reflect in all open tabs, no?